README.rst

```diff
@@ -632,7 +646,7 @@ Create a file named ``.github/workflows/darker.yml`` inside your repository with
              options: "--check --diff --color"
              revision: "master..."
              src: "./src"
-             version: "1.6.0"
+             version: "~=1.6.0"
```

Comment:
This will actually make the GitHub Action install the newest Darker release (before next major release). But will it also cause it to automatically update from time to time?

Reply:
Assuming the action will auto-update, either of the following are good setups:
- pin both Darker and Black to exact versions, or
- pin neither so compatibility updates to Darker will be received if a Black update breaks compatibility.

The bad choice is to pin Darker but let Black update, for obvious reasons.

Should these be clarified in the README?

Reply:
By default there is no darker installed in a GH action environment, correct? In that case pip will install the newest available version (matching the version requirements).
Though people can run actions with a docker image. In such case it would be possible darker is already available. Then the `main.py` will not update Darker, when the current version meets the provided version requirements.

Reply:
But it won't update Black either, so we should be safe except in the unfortunate case where an incompatible Black version was released before we were able to react at all.

Reply:
But the current implementation should downgrade Black, in case Black is incompatible with current Darker version

Reply:
Yes, I believe it will, if we set e.g. `black>=21.5b1,<=22.12.0` as the requirement in the first patch release before we release a fix.